src/app/component-library/catalog/asset-detail-dialog/asset-detail-dialog-data.service.ts

```diff
@@ -26,9 +26,9 @@ export class AssetDetailDialogDataService {
     };
   }
 
-  contractOfferDetails(contractOffer: ContractOffer): AssetDetailDialogData {
-    let asset = contractOffer.asset;
-    let contractPolicy = contractOffer.policy;
+  dataOfferDetails(dataOffer: DataOffer): AssetDetailDialogData {
+    let asset = dataOffer.asset;
+    let contractPolicy = dataOffer.contractOffers[0].policy;
```

Comment:
do they have the same policies for all contractOffers in DataOffer? if that is the case, why we did not add UiPolicy field to DataOffer instead of contractOffer

Reply:
will be fixed in a future PR